matchms/Scores.py

```diff
@@ -121,6 +121,69 @@ def calculate_parallel(self) -> Scores:
         self._scores = self.similarity_function(self.references[:, 0], self.queries[0, :])
         return self
 
+    def scores_by_reference(self, references: ReferencesType) -> numpy.ndarray:
```

Comment:
As function name uses `reference` I expected it would work with a single reference as argument. Why is there support for multiple references? How do you know which score belongs to which reference?

If you want to get the scores for multiple references is not easier to fetch scores for each reference in a loop calling the function. 

I would like to see the function accept only a single reference Spectrum.

Reply:
I changed that.

Reply:
Great